cmsl1t/plotting/efficiency.py

```diff
@@ -52,12 +61,32 @@ def create_histograms(self,
         def make_efficiency(labels):
             this_name = "efficiency" + name.format(**labels)
             this_title = title.format(**labels)
-            eff = asrootpy(ROOT.TEfficiency(this_name, this_title,
-                                            n_bins, low, high))
+            '''Checking type of 'low' to see whether it's int (x-range minimum)
+                    or array (bin edges) for constructing TEfficiency'''
+            if isinstance(low, np.ndarray):
```

Comment:
What about setting high to default to `None` and checking for whether or not it's valid as to what constructor is called?